Multi-World Calendar/script.json

```diff
@@ -1,17 +1,17 @@
 {
-    "$schema": "https://github.com/Roll20/roll20-api-scripts/master/_Example%20Script%20-%20Check%20for%20formatting%20details/script.schema.json",
-    "name": "Multi-World Calendar",
-    "script": "calendar.js",
-    "version": "5.3",
-    "previousversions": ["5.2","5.1","5.0"],
-    "description": "A Calendar for D&D 5e that includes Eberron, Faerun, Greyhawk, Modern and Tal'Dorei",
-    "authors": "Julexar",
-    "patreon": "https://patreon.com/julexar",
-    "roll20userid": "9989180",
-    "modifies": {
-        "state.MWCalendar": "read,write",
-        "state.Alarm": "read,write",
-        "handout": "read,write"
-    },
-    "conflicts": ["Calendar DND", "Calendar"]
+	"$schema": "https://github.com/Roll20/roll20-api-scripts/master/_Example%20Script%20-%20Check%20for%20formatting%20details/script.schema.json",
+	"name": "Multi-World Calendar",
+	"script": "mwcal.js",
+	"version": "1.0",
```

Comment:
This may cause issues, I am going to move forward approving this but in the future I recommend making this a new version, that is incrementally greater than the current version.